### ISSUE TITLE
Error selecting ZMK as currency

### ISSUE THREAD

**Issue by bilbotbaggin** [home-assistant/frontend#14973]

**Comment by frenck** [comment]
`ZMK` is indeed an historical currency, the symbol has been replaced by `ZMW` in 2013.

However, that seems not to be selectable from the Home Assistant frontend. Move the issue to the frontend issue tracker as that is probably a better place in this case.

../Frenck

**Comment by bilbotbaggin** [comment]
Yes that is correct though locally ZMK and ZMW are used interchangably. Though ZMW is the correct currency symbol as you said.
Many thanks for moving my issue to the right place.

Bilbo
